ruby/falcon-sinatra/falcon.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S falcon host
+# frozen_string_literal: true
+
+require "falcon/environment/rack"
+
+hostname = File.basename(__dir__)
+port = ENV["PORT"] || 3000
+
+service hostname do
+  include Falcon::Environment::Rack
+
+  count ENV.fetch("WEB_CONCURRENCY", 1).to_i
+
+  preload "preload.rb"
+
+  # Use HTTP/1.1 for internal backend compatibility with most K8s ingress controllers
```

Comment:
Is it correct that Deploio doesn't support h2c? So that K8 terminates HTTP2 and uses HTTP1.1 at the app itself?

Reply:
Yes, the ingress-controller terminates HTTP2 and uses HTTP1.1 to speak to the deplo.io app.